package.json

```diff
@@ -4,8 +4,9 @@
   "description": "The source files for the We Count website, built with Nuxt.",
   "license": "CC-BY-4.0",
   "contributors": [
-    "Ted Thompson <tthompson@ocadu.ca> (https://github.com/TeddTech)",
+		"Cindy Li <cli@ocadu.ca> (https://github.com/cindyli)",
     "Justin Obara <jobara@ocadu.ca> (https://github.com/jobara)",
+    "Ted Thompson <tthompson@ocadu.ca> (https://github.com/TeddTech)",
     "Ned Zimmerman <nzimmerman@ocadu.ca> (https://github.com/greatislander)"
   ],
   "private": true,
```

Comment:
We've started using an [AUTHORS.md](https://github.com/fluid-project/infusion/blob/master/AUTHORS.md) file for other projects to Indicate those that have copyright. Probably at this point it doesn't matter, but something to think about for later. Also those of us at the IDRC all assign copyright to OCAD University.

Reply:
Added an author field to package.json for now.